### PR TITLE
Feat: Added responsive horizontal card

### PR DESCRIPTION
**Pull request by GauravDe-git**
image

### CODE REVIEW

File: Pages/Card/card.css [hunk 1]
```diff
@@ -54,3 +54,58 @@
     justify-content: space-around;
     padding-bottom: 0.5rem;
 }
+
+.card-horizontal{
+    width: 100%;
+
```
Comment (gadgetfather) [inline Pages/Card/card.css:60]: you can remove extra space here

File: Pages/Card/card.html [hunk 2]
```diff
@@ -9,56 +10,59 @@
...
-    <script>hljs.highlightAll();</script>
+    <script>
+        hljs.highlightAll();
+    </script>
     <script src="https://kit.fontawesome.com/223f475189.js" crossorigin="anonymous"></script>
 </head>
+
 <body>
     <header>
         <h2> <a href="#" class="logo">FlareFrontUI</a></h2>
```
Comment (gadgetfather) [inline Pages/Card/card.html]: Here your href is empty.

File: Pages/Card/card.html [hunk 3]
```diff
@@ -132,11 +141,63 @@ <h3 class="main-content-subheading">eCommerce Card:</h3>
...
+        </code></pre>
+        <h3 class="main-content-subheading">Responsive Horizontal Card:</h3>
+        <div class="component-container">
+            <div class="card-horizontal">
+                <div class="card-horizontal-box">
+                    <div class="card-title">
+                        <h2>Premium Hoody</h2>
+                        <h4>INR 4,900/- </h4>
+                    </div>
+                    <img src="/Assets/imgExample.jpg" alt="">
```
Comment (gadgetfather) [inline Pages/Card/card.html:169]: alt tag is empty here

**Review by gadgetfather — APPROVED** [review]
approving PR with some minor suggestions 👍

**Review by AnkurChunekar — APPROVED** [review]

**Pull request merged**